ui/src/preview/publication-request-dependency-list.tsx

```diff
@@ -29,11 +29,16 @@ const publicationRequestSize = (req: PublishRequestIds): number =>
     req.referenceLines.length +
     req.kmPosts.length;
 
-const TrackNumberItem: React.FC<{ trackNumberId: LayoutTrackNumberId; changeTime: TimeStamp }> = (
-    props,
-) => {
+const TrackNumberItem: React.FC<{
+    layoutContext: LayoutContext;
+    trackNumberId: LayoutTrackNumberId;
+    changeTime: TimeStamp;
+}> = (props) => {
     const { t } = useTranslation();
-    const trackNumbers = useTrackNumbersIncludingDeleted('DRAFT', props.changeTime);
+    const trackNumbers = useTrackNumbersIncludingDeleted(
+        draftLayoutContext(props.layoutContext),
+        props.changeTime,
+    );
```

Comment:
Täällä näkyy hyvin filosofia `LayoutContext`:in passailuun UI-komponentista toiseen. Vaikka komponenttia kiinnostaisikin vain joku tietty konteksti (tässä tapauksessa `DRAFT`-tilainen,) niin passataan silti koko konteksti sellaisenaan ja annetaan komponentin tehdä itse omat päätöksensä